src/SFA.DAS.ProviderApprenticeshipsService.Web/Views/Commitment/Create.cshtml

```diff
@@ -118,9 +127,16 @@
                     <span class="heading-small">£ </span>@Html.TextBoxFor(m => m.Cost, new { @class = "form-control form-control-3-4", type = "text", aria_describedby = "error-message-" + Html.IdFor(m => m.Cost) })
                 </div>
 
+                <div class="form-group @Html.AddClassIfPropertyInError(x => x.ProviderRef, "error")">
+                    @Html.LabelFor(m => m.ProviderRef, "Apprentice provider reference", new { @class = "form-label" })
+                    @Html.ValidationMessageFor(m => m.ProviderRef, null, new { id = "error-message-" + Html.IdFor(m => m.ProviderRef) })
+                    @Html.TextBoxFor(m => m.ProviderRef, new { @class = "form-control form-control-3-4" })
+                </div>
+
                 @Html.HiddenFor(m => m.CommitmentId)
                 @Html.HiddenFor(m => m.ProviderId)
                 @Html.HiddenFor(m => m.AgreementStatus)
+
```

Comment:
I think need to add any field the employer sees but provider doesn't as hidden fields. Or we merge what we have in the UI with whats in the database. We should discuss this with Mark.